actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1272,6 +1273,7 @@ def text_area(object_name, method, options = {})
       # To prevent this the helper generates an auxiliary hidden field before
       # the very check box. The hidden field has the same name and its
       # attributes mimic an unchecked check box.
+      # To disable the hidden field pass +include_hidden: false+ in the options hash.
```

Comment:
Because of the space, this should use `<code>`:

```suggestion
      # To disable the hidden field pass <code>include_hidden: false</code> in the options hash.
```